CWE/events/Italy ahistorical.txt

```diff
@@ -0,0 +1,4166 @@
+ 	id = 8020412
+  	title = "$YEAR$ Italian General Election"
+ 	desc = EVT_8020412_DESC
+  	picture = "ITA_1972_elections"
+	
+  	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1970 
+		NOT = { year = 1990 }
+		
+		government = populist_dictatorship
+		is_vassal = no
+		war = no
+		
+		ruling_party_ideology = conservative 
+		
+		OR = { has_country_flag = ITA_Restoration_Democracy_3 has_country_flag = ITA_Restoration_Democracy_4 }
+		NOT = { has_country_flag = ITA_Democracy_Restored }
+		OR = { has_country_flag = ITA_Coup_fail has_country_flag = ITA_Borghese_dead }
+		has_global_flag = ITA_Operation_Tora_Tora
+	}
+	
+	mean_time_to_happen = {
+		months = 2
+		modifier = {
+			factor = 0.5
+			has_country_flag = ITA_Restoration_Democracy_3
+		}
+	}
+	
+	immediate = { 
+	    government = democracy
+		
+		political_reform = free_opinion
+        political_reform = minorities_oppression
+        political_reform = quota_immigration
+        political_reform = elitist_service
+        political_reform = parties_only_nominated
+        political_reform = lgbt_equal_rights
+        political_reform = responsible_publishing
+        political_reform = no_trade_unions
+        political_reform = proportional_representation
+        political_reform = universal_voting
+        political_reform = free_meetings 
+		political_reform = capital_punishment
+		
+		clr_country_flag = ITA_Coup_fail
+	    clr_country_flag = ITA_Restoration_Democracy_3
+		clr_country_flag = ITA_Restoration_Democracy_4 
+		set_country_flag = ITA_Democracy_Restored 
+	}
+
+	option = {
+		name = "Democratic Opposition (PRI-PSDI-PR)"
+		ruling_party_ideology = liberal
+		
+		any_pop = {
+			ideology = { factor = 0.07 value = big_tent }
+			ideology = { factor = 0.25 value = liberal }
+			ideology = { factor = 0.04 value = progressive }
+			consciousness = -4
+			militancy = -4
+		}
+		
+		set_country_flag = ITA_Democratic_Opposition
+		set_country_flag = ITA_Democracy_Elections2
+		ai_chance = { factor = 0.6 }
+	}
+	
+	option = {
+		name = "Democratic Centre (DC-PLI)"
+		ruling_party_ideology = conservative
+		
+		any_pop = {
+			ideology = { factor = 0.10 value = conservative }
+			ideology = { factor = 0.10 value = big_tent }
+			ideology = { factor = 0.07 value = liberal }
+		}
+		
+		set_country_flag = ITA_Regime_coninuity
+		set_country_flag = ITA_Democracy_Elections
+		ai_chance = { factor = 0.3 }
+	}
+	
+	option = {
+		name = "National Front (MSI-DN-ON-AN)"
+		ruling_party_ideology = nationalist
+		
+		any_pop = {
+			ideology = { factor = 0.20 value = nationalist }
+			ideology = { factor = 0.07 value = conservative }
+			consciousness = 4
+			militancy = 4
+		}
+		
+		set_country_flag = ITA_Regime_coninuity
+		set_country_flag = ITA_Democracy_Elections
+		ai_chance = { factor = 0.1 }
+	}
+	
+}
+
+#Elections 2
+country_event = {
+ 	id = 8020413
+  	title = "$YEAR$ Italian General Election"
+ 	desc = EVT_8020412_DESC
+  	picture = "ITA_1953_elections"
+
```

Comment:
This event must have fire_only_once = yes

Reply:
Hmhhh I didn't add fire_only_once on purpose so that the events can keep to fire every 4 years simulating elections without needing to add more events

Reply:
I see but I think it would be better to keep the event as fire_only_once. Regular elections should be done using the normal system instead so as to not diminish a core feature of the game.